cluster/gce/gci/configure-helper.sh

```diff
@@ -224,6 +224,11 @@ function create-master-auth {
   fi
   local -r basic_auth_csv="${auth_dir}/basic_auth.csv"
   if [[ -n "${KUBE_PASSWORD:-}" && -n "${KUBE_USER:-}" ]]; then
+    if [[ -e "${basic_auth_csv}" && "${METADATA_CLOBBERS_CONFIG:-false}" == "true" ]]; then
+      sed -i '/,admin,system:masters$/d' "${basic_auth_csv}"
+      # The following is for the legacy form of the password line.
+      sed -i '/,admin$/d' "${basic_auth_csv}"
```

Comment:
@lavalamp There are two admin values and the first is actually from expanding the KUBE_USER variable. Happy to make the match cover that string but I believe it should be ,${KUBE_USER:-admin},admin$ if we want to go that route.